### DISCUSSION TITLE
Perbedaan penggunaan untuk crosstab dan pivot

### DISCUSSION THREAD

**Discussion by NiaMiz1** [teamalgoritma/community#818] [Q&A] [answered]
Pertanyaan : 
salam.. 
Selain syntax pivot itu kadang lebih singkat , apakah perbedaan dalam kasus real atau dalam suatu pertanyaan kita menggunakan crosstab atau pivot?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @NiaMiz1 ,
Untuk penggunaan crosstab dan pivot_table tidak terdapat perbedaan yang signifikan. Kedua fungsi tersebut dapat dipilih sesuai preferensi dan kenyamanan Anda. Akan tetapi, dari segi versatility dan kecepatan, pivot lebih baik daripada crosstab. Perbedaan yang lain juga, pivot menerima input dalam bentuk dataframe dan crosstab menerima input dalam bentuk array. Detail lebih lanjut dapat dilihat pada dokumentasi berikut:

- [Dokumentasi CrossTab](https://pandas.pydata.org/docs/reference/api/pandas.crosstab.html).
- [Dokumentasi Pivot Table](https://pandas.pydata.org/docs/reference/api/pandas.pivot_table.html).

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma
